src/connection/serverComponent.ts

```diff
@@ -115,9 +120,24 @@ export class ServerComponent {
         const lastInstalledName = Config.getServerComponentName();
 
         if (lastInstalledName !== basename || this.installed === false) {
-          const updateQuestion = await window.showInformationMessage(`An update to the database server component is required: ${basename}`, `Update`);
+          let doUpdate = false;
```

Comment:
Should this be something like?:
```javascript
let doUpdate = shouldAutoUpdate();
```
(current implementation may be preferred for readability with subsequent logic, so no assertions here)